### PR TITLE
PACA-1085 - Partially Adjudicated Claims Data website updates

### PR DESCRIPTION
**Pull request by daryll**
## 🎫 Ticket

https://jira.cms.gov/browse/PACA-1085

## 🛠 Changes

-  add/update BCDA Partially Adjudicated Claims Data info
-  add BCDA partially adjudicated data dictionary
-  add FAQ

## ℹ️ Context for reviewers

- relevant info for customers for the Full Launch of the BCDA Partially Adjudicated Enhancement

## ✅ Acceptance Validation

- all text was verified and approved by CMS and BCDA, review links below for more details:
  - https://confluence.cms.gov/x/2oJUMQ
  - https://confluence.cms.gov/x/DpPKGw
  - https://confluence.cms.gov/x/3gSPMQ

## 🔒 Security Implications

- [ ] New data is stored or transmitted.
  

- [ ] This change introduces new software dependencies.
  

- [ ] This change impacts security controls or alters supporting software.
  

- [ ] A [security checklist](https://confluence.cms.gov/display/BCDA/Security+Checklists) is completed for this change.
  

- [ ] More team discussion required to evaluate security implications.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ global_nav:
     id: data
   - title: Updates
     id: updates
+  - title: BCDA Partial Adjudicated Claims
```
Comment (greg-pf) [inline _config.yml:39]: Should this read as "Partially Adjudicated Claims"?
```suggestion
  - title: BCDA Partially Adjudicated Claims
```

File: _includes/guide/try_the_api.html [hunk 2]
```diff
@@ -44,15 +44,14 @@ <h3>
 </div>
 
 <h3>
-	BCDA Partially Adjudicated Enhancement  - Two Sizes
+  BCDA Partially Adjudicated Claims
```
Comment (greg-pf) [inline _includes/guide/try_the_api.html:47]: Suggest the following to be consistent with the titles of the datasets above
```suggestion
  Partially Adjudicated Claims Datasets - Two Sizes
```

File: _includes/guide/bcda_enhancement_credentials.html
Comment (greg-pf) [inline _includes/guide/bcda_enhancement_credentials.html:1]: I don't believe we should revert back to the use of "DCE", the GPDC Model was replaced with ACO REACH. The term used for organizations participating in ACO REACH is "REACH ACO". Can you confirm what term we should be using here?

File: _includes/guide/try_the_api.html (hunk 2 above)
Comment (greg-pf) [inline _includes/guide/try_the_api.html:47]: Suggest the following to be consistent with the titles of the datasets above
```suggestion
  Partially Adjudicated Claims Datasets - Two Sizes
```

File: partial.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+id: partial
+title: BCDA Partial Adjudicated Claims
+layout: default
+---
+<div class="ds-base ds-l-container">
+  <div role="region" aria-label="page-title" class="ds-l-row ds-u-fill--primary-darker">
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
+      BCDA Partial Adjudicated Claims
+    </div>
```
Comment (greg-pf) [inline partial.html:10]: Same as earlier question, should this be "Partially Adjudicated"?
```suggestion
id: partial
title: BCDA Partially Adjudicated Claims
layout: default
---
<div class="ds-base ds-l-container">
  <div role="region" aria-label="page-title" class="ds-l-row ds-u-fill--primary-darker">
    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
      BCDA Partially Adjudicated Claims
    </div>
```

**Review by karinamzalez — DISMISSED** [review]
Agree with greg's comment's but otherwise -- lgtm!

File: _config.yml (hunk 1 above)
Comment (daryll) [reply _config.yml:39]: pls refer to the approved doc from CMS
https://confluence.cms.gov/x/2oJUMQ

File: _includes/guide/try_the_api.html (hunk 2 above)
Comment (daryll) [reply _includes/guide/try_the_api.html:47]: pls refer to the approved doc from CMS
https://confluence.cms.gov/x/2oJUMQ

Comment (daryll) [reply _includes/guide/try_the_api.html:47]: pls refer to the approved doc from CMS
https://confluence.cms.gov/x/2oJUMQ

**Comment by greg-pf** [comment]
Looks good to me 👍  Thanks Daryll!

**Review by karinamzalez — APPROVED** [review]

**Pull request merged**